### ISSUE TITLE
Incomplete wheel

### ISSUE THREAD

**Issue by FrBrGeorge** [CSRedRat/topalias#12]
* topalias-2.0.8
* Python 3.8.6
* ALT Linux Sisyphus 2020-11-19

### Description

When installing topalias with `pip3 install -U --user topalias`, the `topalias` script drops an exception not founding `data/.bash_aliases` and exits instead of working properly.

My guess that is because of PyPi wheel version of topalias doesn't include any data directory.

### What I Did

```
$ python3 -m venv tmp/topa
$ cd tmp/topa
$ . bin/activate
(topa) $ pip install topalias
(topa) $ topalias
…
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/tmp/topa/topalias/data/.bash_aliases'
(topa) $ pip download topalias
…
(topa) $ zipinfo topalias-2.0.8-py2.py3-none-any.whl
Archive:  topalias-2.0.8-py2.py3-none-any.whl
Zip file size: 10494 bytes, number of entries: 11
-rw-r--r--  2.0 unx      385 b- defN 20-Nov-19 05:26 topalias/__init__.py
-rw-r--r--  2.0 unx      318 b- defN 20-Nov-19 05:26 topalias/__main__.py
-rw-r--r--  2.0 unx     7671 b- defN 20-Nov-19 05:26 topalias/aliascore.py
-rw-r--r--  2.0 unx     3986 b- defN 20-Nov-19 05:26 topalias/cli.py
-rw-r--r--  2.0 unx      918 b- defN 20-Nov-19 05:26 topalias/statistic.py
-rw-rw-rw-  2.0 unx     1605 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/LICENSE
-rw-r--r--  2.0 unx     6838 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/METADATA
-rw-r--r--  2.0 unx      110 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/WHEEL
-rw-r--r--  2.0 unx       47 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/entry_points.txt
-rw-r--r--  2.0 unx        9 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/top_level.txt
?rw-rw-r--  2.0 unx      866 b- defN 20-Nov-19 05:27 topalias-2.0.8.dist-info/RECORD
11 files, 22753 bytes uncompressed, 9032 bytes compressed:  60.3%

```

**Comment by CSRedRat** [comment]
Please test:
`pip install -i https://test.pypi.org/simple/ topalias`
From: https://test.pypi.org/project/topalias

**Comment by FrBrGeorge** [comment]
Now the wheel lacks `bin/topalias` script (you've missed entry point, I think).
Anyway, as for now, your program searches config files __only in__ `./topalias/data`, not in the python's library directory, where they're placed:
```
(click) george@inspiron:~/venv.../b/c/d> pwd
/home/george/venv/_click/a/b/c/d
(click) george@inspiron:~/venv.../b/c/d> strace -efile topalias |& grep bash_aliases 
stat("./.bash_aliases", 0x7fff450c7c70) = -1 ENOENT (Нет такого файла или каталога)
stat("/home/george/.bash_aliases", 0x7fff450c7c70) = -1 ENOENT (Нет такого файла или каталога)
stat("./.bash_aliases", 0x7fff450c7a90) = -1 ENOENT (Нет такого файла или каталога)
stat("/home/george/.bash_aliases", 0x7fff450c7a90) = -1 ENOENT (Нет такого файла или каталога)
openat(AT_FDCWD, "/home/george/venv/_click/a/b/c/d/topalias/data/.bash_aliases", O_RDONLY|O_CLOEXEC) = -1 ENOENT (Нет такого файла или каталога)
File .bash_aliases not found in any of the directories
File .bash_aliases not found in any of the directories
FileNotFoundError: [Errno 2] No such file or directory: '/home/george/venv/_click/a/b/c/d/topalias/data/.bash_aliases'
```

**Comment by zabuldon** [comment]
@CSRedRat The issue was caused by the https://github.com/CSRedRat/topalias/blob/e9105abed3041ca2ca8fd5d4a6d41917f26aae0b/topalias/aliascore.py#L66
you just quoted __file__ variable. 

import os
print(os.path.dirname(os.path.realpath("__file__")))
print(os.path.dirname(os.path.realpath(__file__)))


Example output:
/Users/sergey.isachenko/Work
/Users/sergey.isachenko/

**Comment by CSRedRat** [comment]
fix path definition in 2.0.11, please update:
```
pip3 install -U --user topalias
# or
pip3 install -U --user topalias==2.0.11
```

**Comment by FrBrGeorge** [comment]
Now as any use of `/data/` is gone, the bug isn't actual anymore.
